angularjs-portal-frame/src/main/webapp/portal/notifications/partials/notifications-full.html

```diff
@@ -0,0 +1,16 @@
+<div class="row portlet-frame">
+  <div class="portlet-header">
+    <img class="header-image" ng-src="{{$storage.homeImg}}" alt="UW Campus cover image">
+    <h1>Notifications</h1>
+  </div>
+  <div class="col-xs-12">
+    <ul class="notifications-list" ng-controller="NotificationController as notificationCtrl">
+      <!--TODO <li class="notification-item" ng-repeat="notification in notificationCtrl.notifications"
+-          ng-class="notificationCtrl.getClass($index, notificationCtrl.notifications)"
+-          ng-click="notificationCtrl.markRead($index, notificationCtrl.notifications)"> -->
+      <li class="notification-item" ng-repeat="notification in notificationCtrl.notifications">
+        <a ng-href='{{notification.actionURL}}' target='_blank' aria-label='notification.actionAlt'>{{ notification.title }}</a>
```

Comment:
This is an interesting use of `aria-label` : does it help screen readers to have this alternative label, or is the `notification.title` already sufficient label?